examples/angular/src/pages/ui/components/authenticator/custom-sign-up-fields/custom-sign-up-fields.component.html

```diff
@@ -4,13 +4,6 @@
     amplifySlot="sign-up-form-fields"
     let-validationErrors="validationErrors"
   >
-    <!--  Prepend `preferred_username` custom attribute  -->
-    <amplify-text-field
```

Comment:
can be deleted as it's already inferred from zero config

Reply:
👍